micro_sam/training/training.py

```diff
@@ -157,6 +174,7 @@ def train_sam(
     pbar_signals: Optional[QObject] = None,
     optimizer_class: Optional[Optimizer] = torch.optim.AdamW,
     peft_kwargs: Optional[Dict] = None,
+    verify_n_labels_in_loader: Optional[int] = None,
```

Comment:
I would set the default to something rather small (maybe 50). Otherwise `None` causes quite long wait times for a big data loader.

Reply:
Yep, that sounds reasonable. I've updated the defaults now to `50`. Thanks!